src/utils/getAge.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (dateString) => {
```

Comment:
please describe what tyope of datestring is allowed as input here

Reply:
I do not know. :D 
It is only moved from hooks to utils.